test/modules/MetricsValidator/src/Program.cs

```diff
@@ -37,10 +37,19 @@ static async Task<int> MainAsync()
                     .AddEnvironmentVariables()
                     .Build();
 
+                if (Environment.OSVersion.Platform != PlatformID.Unix)
+                {
+                    Logger.LogInformation("Windows detected, delaying");
```

Comment:
We want to be _very_ tidy and concise in the information-level logs. So either remove these `LogInformation` statements or at least move them to `LogDebug`.